app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -58,6 +60,11 @@ def get_network_interfaces
         @network_interfaces = gather_data_for_this_region(@nis)
       end
 
+      # Can use any service object here, results are identical.
+      def get_subscriptions
+        @subscriptions = @vmm.subscriptions.map(&:subscription_id)
```

Comment:
@bzwei A client might expect it, though. Anyway, I've updated the code to use get_subscriptions instead of @subscriptions. This one extra call shouldn't be a concern. We can always review later.